force_wfmanager/left_side_pane/workflow_tree.py

```diff
@@ -32,7 +33,12 @@
 # Actions!
 new_mco_action = Action(name='New MCO...', action='new_mco')
 delete_mco_action = Action(name='Delete', action='delete_mco')
-edit_mco_action = Action(name='Edit...', action='edit_mco')
+edit_mco_action = Action(name='Edit...', action='edit_mco',
+                         enabled_when='handler.mv_editable(object)')
```

Comment:
in this case you are forced to use object, because it's the way it was designed (likely before the object class was introduced, some of our stuff is really ancient)